product-database/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"log"
+	"os"
+)
+
+var Item []struct {
```

Comment:
Given that this represents a _list_ of things, a plural name would be best. 
But why an anonymous struct though? Why not have a `Product` or `product` custom type?